File: iact-check-webapp/src/main/resources/webapp/src/app/admin/shared/dtos/email-settings-dto.ts
export interface EmailSettingsDTO {
  sendEmails: boolean;
  smtpHost: string;
  smtpPort: number;
  smtpUsername: string;
  smtpPassword: string;
  smtpTransportStrategy: string;
  fromAddress: string;
  formName: string;
}

File: iact-check-core/src/main/kotlin/ch/iact/iactcheck/dto/EmailSettingsDTO.kt
package ch.iact.iactcheck.dto

data class EmailSettingsDTO(
    val sendEmails: Boolean,
    val smtpHost: String,
    val smtpPort: Int,
    val smtpUsername: String,
    val smtpPassword: String,
    val smtpTransportStrategy: SmtpTransportStrategy,
    val fromAddress: String,
    val fromName: String
)
